app/controllers/admin/dashboards_controller.rb

```diff
@@ -1,17 +1,19 @@
-class Admin::DashboardsController < Admin::ApplicationController
-  def index
-    @slide_count = Slide.count
-    @user_count = User.count
-    @conversion_failed_count = Slide.failed.count
-    @comment_count = Comment.count
+module Admin
+  class DashboardsController < Admin::BaseController
+    def index
```

Comment:
__[rails_best_practices]__
* move model logic into model (Slide use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)

<!-- Comment by [SideCI](https://sideci.com) -->